src/ui/features/onboarding/shared/SessionExpired.tsx

```diff
@@ -28,11 +28,11 @@ export function SessionExpired({ onSubmit }: { onSubmit(): void }) {
             Session expired
           </UIText>
           <UIText kind="body/regular" color="var(--neutral-600)">
-            Try create another wallet
+            Try creating another wallet
```

Comment:
👍

Reply:
Updated to "Try creating or importing another wallet"